tools/generate_data_static.py

```diff
@@ -0,0 +1,320 @@
+            if repo_url:
+                item["repo_url"] = repo_url
+
+        # Handle logo
+        logo_url = proj.get("logo")
+        if logo_dir is not None and logo_url:
+            # Download logo and use only the file name in YAML
+            file_name = download_logo(logo_url, logo_dir)
+            item["logo"] = file_name
+        else:
+            # Without download, keep full URL or fallback
+            if logo_url:
+                item["logo"] = logo_url
+            else:
+                item["logo"] = "placeholder.svg"
+
+        # Append item to subcategory
+        subcat["items"].append(item)
+
+    # Convert nested dict of subcategories to list structure required by YAML
+    category_list = []
+    for cat in categories.values():
+        subcat_list = []
+        for subcat in cat["subcategories"].values():
+            subcat_list.append(subcat)
+        cat["subcategories"] = subcat_list
+        category_list.append(cat)
+
+    return {"categories": category_list}
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser(description="Generate data.yml for landscape2")
+    parser.add_argument(
+        "--input",
+        help="Path to a local JSON file containing project data (optional)",
+    )
+    parser.add_argument(
+        "--output",
+        default="data.yml",
+        help="Name of the YAML file to generate (default: data.yml)",
+    )
+    parser.add_argument(
+        "--categories",
+        default="static_categories.yml",
+        help=(
+            "Path to a YAML file defining static categories. If provided, the script"
+            " will organise projects according to this file. If not present,"
+            " projects are grouped using their 'category' field."
+        ),
+    )
+    args = parser.parse_args()
+
+    # Load projects
+    if args.input:
+        projects = load_projects_from_file(Path(args.input))
+    else:
+        projects = fetch_projects_from_api()
+
+    # Download logos into a local 'logos' directory and reference them in YAML
+    logo_dir = Path("logos")
+    # If a categories file exists, load it; otherwise use dynamic grouping
+    categories_path = Path(args.categories)
+    if categories_path.exists():
+        static_categories = load_static_categories(categories_path)
+        landscape_data = build_landscape_from_static(
+            projects, static_categories, logo_dir=logo_dir
+        )
+    else:
+        # Fall back to dynamic grouping if no categories file is provided
+        landscape_data = build_landscape_from_dynamic(
+            projects, logo_dir=logo_dir
+        )
+
+    # Write YAML file
+    out_path = Path(args.output)
+    with out_path.open("w", encoding="utf-8") as f:
+        yaml.dump(
+            landscape_data,
+            f,
+            sort_keys=False,
+            allow_unicode=True,
+            default_flow_style=False,
+        )
+    print(f"Generated {out_path}")
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
New line at end of file

```suggestion
    main()

```